src/components/Reusables/ConfirmationPage/ConfirmationPage.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Page,
+  PageContent,
+  PageHeader,
+  List,
+  BorderStyle,
+  PageColor,
+} from '@altinn/altinn-design-system';
+import type { Key } from 'react';
+
+import type { DelegableApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { useAppDispatch } from '@/rtk/app/hooks';
+import type { DelegableOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { ReactComponent as OfficeIcon } from '@/assets/Office1.svg';
+import { ReactComponent as SettingsIcon } from '@/assets/Settings.svg';
+import { CompactDeletableListItem } from '@/components/Reusables/CompactDeletableListItem';
+
+import classes from './ConfirmationPage.module.css';
+
+export interface ConfirmationPageProps {
+  firstListItems: DelegableApi[];
+  secondListItems: DelegableOrg[];
+  headerText: string;
+  firstContentText?: string;
+  secondContentText?: string;
+  bottomText?: string;
+  mainButton?: React.ReactNode;
+  complementaryButton?: React.ReactNode;
+  headerIcon: React.ReactNode;
+  color?: PageColor;
+}
+
+export const ConfirmationPage = ({
+  firstListItems,
+  secondListItems,
+  headerText,
+  firstContentText,
+  secondContentText,
+  bottomText,
+  mainButton,
+  complementaryButton,
+  headerIcon,
+  color = PageColor.Primary,
+}: ConfirmationPageProps) => {
+  const dispatch = useAppDispatch();
+
+  const apiListItems = firstListItems.map((api: DelegableApi, index: Key) => {
+    return (
+      <CompactDeletableListItem
+        key={index}
+        startIcon={<SettingsIcon />}
+        removeCallback={firstListItems.length > 1 ? () => dispatch(softRemoveApi(api)) : null}
+        firstText={api.apiName}
+        secondText={api.orgName}
+      ></CompactDeletableListItem>
+    );
+  });
+
+  const orgListItems = secondListItems.map((org: DelegableOrg, index: Key | null | undefined) => {
+    return (
+      <CompactDeletableListItem
+        key={index}
+        startIcon={<OfficeIcon />}
+        removeCallback={secondListItems.length > 1 ? () => dispatch(softRemoveOrg(org)) : null}
+        firstText={org.orgName}
+        secondText={org.orgNr}
+      ></CompactDeletableListItem>
+    );
+  });
+
+  return (
+    <div>
+      <div className={classes.page}>
+        <Page color={color}>
+          <PageHeader icon={headerIcon}>{headerText}</PageHeader>
+          <PageContent>
+            <div className={classes.pageContent}>
+              <h2>{firstContentText}</h2>
+              <List borderStyle={BorderStyle.Dashed}>{apiListItems}</List>
+              <h2 className={classes.secondText}>{secondContentText}</h2>
+              <List borderStyle={BorderStyle.Dashed}>{orgListItems}</List>
+              <h3 className={classes.bottomText}>{bottomText}</h3>
+              <div className={classes.navButtonContainer}>
```

Comment:
Why are the buttons placed in the middle of the page?
![image](https://user-images.githubusercontent.com/22012997/208307254-bc554f9f-4abf-4b3e-acd2-3352ff21f369.png)

Reply:
Hanne wanted them there

Reply:
But the design isn't like that though. The design has them on the left (which I also disagree with for that matter, since I think they should be on the right, but that's another question entirely)

Reply:
But if Hanne has approved this, then I suppose it's right

Reply:
Oh, but one things though:
Maybe add some more space under the buttons? (And maybe move them up a smidge?)

Right now it looks a bit cramped and doesn't follow framing rules.
(As I look at the design sketches I notice there's more space there too)
![image](https://user-images.githubusercontent.com/22012997/208412042-9688f69c-f3c0-4994-8b85-f4ad8428c0bc.png)

Reply:
Aaaahh, that makes sense. It's all good then 😊